packages/http-server/README.md

```diff
@@ -9,7 +9,7 @@ import { createServer } from '@stoplight/prism-http-server';
 
 const server = createServer({
   path: './api.oas2.json',
```

Comment:
This part is outdated. `createServer` signature is `const createServer = (operations: IHttpOperation[], opts: IPrismHttpServerOpts): IPrismHttpServer`.